assets/aws/go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/gravitational/teleport-ami-update
+
+go 1.19
+
+require (
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137 // indirect
+	github.com/aws/aws-sdk-go-v2 v1.17.3 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.27 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.21 // indirect
+	github.com/aws/aws-sdk-go-v2/service/dynamodb v1.17.9 // indirect
+	github.com/aws/aws-sdk-go-v2/service/ec2 v1.77.0 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.9.11 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/endpoint-discovery v1.7.21 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.9.21 // indirect
+	github.com/aws/smithy-go v1.13.5 // indirect
+	github.com/gravitational/kingpin v2.1.10+incompatible // indirect
```

Comment:
why are all of these deps indirect? confused me so I cloned locally and `go mod tidy` creates a significant diff

Reply:
Thanks, good catch!